docs/sources/alerting/fundamentals/notifications/index.md

```diff
@@ -11,19 +12,37 @@ keywords:
   - contact point
   - notification channel
   - create
-title: Contact points
+title: Notifications
 weight: 410
 ---
 
-# Contact points
+# Notifications
 
-Use contact points to define how your contacts are notified when an alert rule fires. A contact point can have one or more contact point types, for example, email, slack, webhook, and so on. When an alert rule fires, a notification is sent to all contact point types listed for a contact point. Contact points can be configured for the Grafana Alertmanager as well as external alertmanagers.
+Notifications are sent when an alert is firing or has been resolved, and configuring how and where a notification should be sent is done via notification policies. With notification policies it is possible to configure how often a notification should be sent; and whether alerts should all be sent in the same notification, sent in grouped notifications based on a set of labels, or as seperate notifications.
 
-You can also use message templating to customize notification messages for contact point types.
+## Notification policies
 
-## Supported contact point types
+Notification policies control when and where notifications are sent. A notification policy can choose to send all alerts together in the same notification, send alerts in grouped notifications based on a set of labels, or send alerts as separate notifications. Each notification policy can have its own timings configuration that control how often notifications should be sent; and can reference one or more mute timings to inhibit notifications at certain times of the day and on certain days of the week.
```

Comment:
This contains a lot of the text from above - you could include the last part in the paragraph above?

Reply:
Also what about silences?